src/sentry/static/sentry/app/views/settings/components/settingsNavigationGroup.jsx

```diff
@@ -48,13 +55,21 @@ export default class NavigationGroup extends React.Component {
           });
 
           return (
-            <SettingsNavItem
-              key={title}
-              to={to}
-              label={title}
-              index={index}
-              badge={badgeResult}
-            />
+            <StyledItem key={title}>
+              <SettingsNavItem
+                key={title}
+                to={to}
+                label={title}
+                index={index}
+                badge={badgeResult}
+              />
+              <Hook
+                name="sidebar:secondary-icon"
```

Comment:
Same idea here as above for hooking into nav items, but lets call it `settings-nav:item-label`